central/scannerdefinitions/handler/updater_test.go

```diff
@@ -39,7 +39,7 @@ func TestUpdate(t *testing.T) {
 	require.NoError(t, u.doUpdate())
 	assertOnFileExistence(t, filePath, true)
 
-	lastUpdatedTime := time.Now().Add(time.Hour)
```

Comment:
I think the CDN should return the expected result, even with a time that's in the future; however, I'd rather unblock CI.